setup.py

```diff
@@ -6,314 +6,5 @@
 # License: 3-clause BSD.  The full license text is available at:
 #  - https://github.com/uqfoundation/dill/blob/master/LICENSE
 
-import os
-
-# set version numbers
-stable_version = '0.2.9'
-target_version = '0.3.0'
-is_release = stable_version == target_version
-
-# check if easy_install is available
-try:
-#   import __force_distutils__ #XXX: uncomment to force use of distutills
-    from setuptools import setup
-    has_setuptools = True
-except ImportError:
-    from distutils.core import setup
-    has_setuptools = False
-
-# generate version number
```

Comment:
Admittedly ugly (and hackish code from an old version of `numpy`) primarily to manage version number and docs.  I agree, it does need a change.